src/index.ts

```diff
@@ -73,4 +76,31 @@ const mimeRendererPlugin: JupyterFrontEndPlugin<void> = {
   }
 };
 
-export default [plugin, executorPlugin, mimeRendererPlugin];
+const bibPlugin: JupyterFrontEndPlugin<IBibliographyManager> = {
+  id: 'jupyterlab-myst:bibliography',
+  requires: [],
+  provides: IBibliographyManager,
+  autoStart: true,
+  activate: (app: JupyterFrontEnd) => {
+    console.log('Using jupyterlab-myst:bibliography');
+
+    const bibFile = 'bibliography.bib';
+    const manager = new BibliographyManager(
+      app.serviceManager.contents,
+      bibFile
+    );
+    manager.changed.connect((manager, renderer) => {
+      console.log(renderer, 'CHANGE');
+      // TODO: not sure how to pass this state over to the myst renderer. We need some global state?
+      // If that is the case, we can do that using redux.
+      if (renderer) {
+        citationRenderers[bibFile] = renderer;
+      } else {
+        delete citationRenderers[bibFile];
+      }
```

Comment:
When the manager changes, we just update the shared state, which is then available next time we render markdown. I don't really think we need to go full: update-the bib when the markdown changes as it could potentially render every single cell.